src/errors/error_handler.js

```diff
@@ -11,8 +11,30 @@ import InternalServerError from './api/internal_server_error';
 import ServiceUnavailableError from './api/service_unavailable_error';
 import GatewayTimeoutError from './api/gateway_timeout_error';
 import ForbiddenError from './api/forbidden_error';
+import EasyPostError from './easypost_error';
+import Constants from '../constants';
 
 export default class ErrorHandler {
+  /**
+   * Recursively traverses a JSON object or array and extracts error messages
+   * as strings. Adds the extracted messages to the specified messagesList array.
+   *
+   * @param {object|array|string} errorMessage - The JSON object or array to traverse.
+   * @param {array} messagesList - The array to which extracted error messages will be added.
+   */
+  static traverseJsonElement(errorMessage, messagesList) {
+    if (errorMessage instanceof Object) {
+      for (const value of Object.values(errorMessage)) {
```

Comment:
suggestion: we use `value` in one block and `element` in the very next. Let's have them both be the same name

Reply:
👌